ruby_on_rails/advanced_forms_and_activerecord/active_record_queries.md

```diff
@@ -220,8 +220,8 @@ Sometimes, you just can't get ActiveRecord to do what you want it to.  In that c
 ### Querying basics
 1. Read the first 6 sections of the [Rails Guide on Active Record Querying](http://guides.rubyonrails.org/active_record_querying.html) for a more basic overview of query functions. Don't worry too much about batching and `#find_each`.
 2. Read section 21 of the [same Rails Guide](https://guides.rubyonrails.org/active_record_querying.html#existence-of-objects) for a brief look at using `exists?`, `any?` and `many?`.
-3. Read sections 7, 8 and 22 of the [same Rails Guide](https://guides.rubyonrails.org/active_record_querying.html#group) for an understanding of aggregate functions and the calculations you can run on them.
-4. Skim sections 9-12 of the [same Rails Guide](https://guides.rubyonrails.org/active_record_querying.html#overriding-conditions).  
+3. Read sections 7, 8 and 22 of the [same Rails Guide](https://guides.rubyonrails.org/active_record_querying.html#grouping) for an understanding of aggregate functions and the calculations you can run on them.
+4. Skim sections 9-12 of the [same Rails Guide](https://guides.rubyonrails.org/active_record_querying.html#null-relation).  
```

Comment:
We don't want to change the link here. `overriding-conditions` is where we want to link to on the page but what we need to do is change the section numbers to reflect that. From looking at the new sections I think it might just be section 8 now that covers this but we should probably keep 9 as well.

Reply:
Okay, i replaced #null-relation back to #overriding-conditions. You'll let me know if any changes are required.